backstop/tests/breadcrumbs.html

```diff
@@ -55,155 +54,324 @@
+            </button>
+          </li>
+          <li
+            class="iui-breadcrumbs-separator"
+            aria-hidden
+          >
+            <svg-chevron-right aria-hidden></svg-chevron-right>
+          </li>
+          <li class="iui-breadcrumbs-item">
+            <span class="iui-breadcrumbs-text">&mldr;</span>
+          </li>
+          <li
+            class="iui-breadcrumbs-separator"
+            aria-hidden
+          >
+            <svg-chevron-right aria-hidden></svg-chevron-right>
+          </li>
+          <li class="iui-breadcrumbs-item">
+            <button
+              class="iui-button"
+              id="test-3"
             >
-              <svg-chevron-right aria-hidden="true"></svg-chevron-right>
-            </li>
-            <li class="iui-breadcrumbs-item">
-              <a
-                href="#"
-                id="test-2"
-              >Double nested folder with truncation</a>
-            </li>
-            <li
-              class="iui-breadcrumbs-separator"
-              aria-hidden
+              <span class="iui-button-label">Nested</span>
+            </button>
+          </li>
+          <li
+            class="iui-breadcrumbs-separator"
+            aria-hidden
+          >
+            <svg-chevron-right aria-hidden></svg-chevron-right>
+          </li>
+          <li class="iui-breadcrumbs-item">
+            <button
+              class="iui-button"
+              id="test-4"
             >
-              <svg-chevron-right aria-hidden="true"></svg-chevron-right>
-            </li>
-            <li class="iui-breadcrumbs-item iui-current">
-              <span aria-current='page'>You are here</span>
-            </li>
-          </ol>
-        </nav>
-
-      </section>
-
-      <section id="demo-buttons">
-
-        <nav
-          class="iui-breadcrumbs"
-          aria-label="Breadcrumbs"
-        >
-          <button class="iui-button iui-borderless iui-dropdown">
-            <svg-folder
-              class="iui-button-icon"
-              aria-hidden="true"
-            ></svg-folder>
-            <svg-caret-down-small
-              class="iui-button-icon"
-              aria-hidden="true"
-            ></svg-caret-down-small>
-          </button>
-          <ol class="iui-breadcrumbs-list">
-            <li class="iui-breadcrumbs-item">
-              <button class="iui-button">
-                <span class="iui-button-label">Root</span>
-              </button>
-            </li>
-            <li
-              class="iui-breadcrumbs-separator"
-              aria-hidden
+              <span class="iui-button-label">Double nested folder with truncation</span>
+            </button>
+          </li>
+          <li
+            class="iui-breadcrumbs-separator"
+            aria-hidden
+          >
+            <svg-chevron-right aria-hidden></svg-chevron-right>
+          </li>
+          <li
+            class="iui-breadcrumbs-item"
+            aria-current
```

Comment:
so li element shouldnt get `aria-current`. Child element would, right? remove this

Reply:
Moved `aria-current` to the button, no longer on the `<li>`